data/card_catalogue.json

```diff
@@ -1358,27 +1320,27 @@
         {
           "title": "Wind Storm",
           "upgrade": "Hurricane",
-          "tier": "green",
+          "tier": "red",
           "details": {
             "rules": "Knock any number of visible targets airborne. If any are already already airborne, knock them back 10 squares",
-            "flavour": "Life isn't about waiting for the storm to pass...It's about learning to dance in the rain.",
+            "flavour": "Rise, rise towards the heavens.",
             "image": "assets/core/card/illustration/Wind Storm.png"
           }
         },
         {
           "title": "Hurricane",
           "upgrade": "",
-          "tier": "red",
+          "tier": "gold",
           "details": {
-            "rules": "Knock any number of visible targets airborne.  For the rest of combat, characters who become airborne lose their next turn.",
-            "flavour": "Rise, rise towards the heavens",
+            "rules": "A powerful hurricane is summoned over the play area which lasts as long as you want.",
```

Comment:
Making this more like Tsunami, a map shattering effect. This one is less immediately devastating but permanent